deploy/2-eks-cluster-with-import-vpc/eks/main.tf

```diff
@@ -35,14 +35,39 @@ terraform {
   }
 }
 
+data "aws_region" "current" {}
+
+data "aws_availability_zones" "available" {}
+
+data "aws_eks_cluster" "cluster" {
+  name = module.aws-eks-accelerator-for-terraform.eks_cluster_id
+}
```

Comment:
I changed this file heavily with my new PR. May be you can revoke the change to this file.